server/api/operator.go

```diff
@@ -79,7 +92,7 @@ func (h *operatorHandler) List(w http.ResponseWriter, r *http.Request) {
 		err     error
 	)
 
-	kinds, ok := r.URL.Query()["kind"]
+	kinds, ok := r.URL.Query()[pKind]
```

Comment:
Will this have compatibility issues?

Reply:
First of all, this PR we only adjust(change) the response body. 
And I'm not sure if it is ok haha. 
However, the error response for invalid or missing parameters before seems defined casually,  examples :
- (region_id)`region_ids is invalid`
- (region_id)`missing region id`
- (to_store_id) `"missing store id to transfer leader to"`
- (from_store_id) `"invalid store id to transfer peer from"`

in this PR there were changed to 
- (region_id) `missing region_id`
- (to_store_id) `missing to_store_id`
- (from_store_id) `invalid from_store_id`

Thus the response is totally different, I'm not sure if it is ok, any suggestion?